.github/actions/mutation-tests-changed-files/action.yml

```diff
@@ -0,0 +1,96 @@
+name: Run mutation tests for changed files only 
+inputs:
+  package-name:
+    description: 'Name of the package'
+    required: true
+  stryker_dashboard_api_key:
+    description: 'Stryker dashboard api key'
+    required: true
+runs:
+  using: "composite"
+  steps:
+      - uses: actions/setup-node@v3
+        with:
+          node-version-file: .nvmrc
+
+      - name: Setup environment
+        uses: ./.github/actions/setup
+
+      - name: Get changed files
+        uses: tj-actions/changed-files@v39.0.0
+        id: changed-files
+        with:
+          files_yaml: |
+            src:
+              - 'src/**/*.ts'
+              - '!src/**/*.spec.ts'
+              - '!src/**/__mocks__/**'
+          separator: "','"
+          path: packages/${{ inputs.package-name }}
+
+      - name: Run mutation tests
+        if: steps.changed-files.outputs.src_any_changed == 'true'
+        shell: bash
+        id: run-mutation-tests
+        run: yarn workspace @editorjs/${{ inputs.package-name }} test:mutations --mutate ${{format('''{0}''', steps.changed-files.outputs.src_all_changed_files)}}
+        continue-on-error: true
+
+      - name: Find current PR's number
+        uses: jwalton/gh-find-current-pr@v1
+        id: findPr
+
+      - name: Comment on successful mutation testing
+        uses: thollander/actions-comment-pull-request@v2
+        env:
+          STRYKER_DASHBOARD_API_KEY: ${{ inputs.stryker_dashboard_api_key }}
+        if: steps.changed-files.outputs.src_any_changed == 'true' && steps.run-mutation-tests.outcome == 'success' && ${{ steps.findPr.outputs.number != '' }}
+        with:
+          message: |
+            ## ✅ Mutation testing passed
```

Comment:
```suggestion
            ✅ Mutation testing passed
```

Reply:
Why? It just makes the line bold

Reply:
it makes a big header, but it is actually a simple message, like regular comment. Emoji is used for high-speed determination, but big headers just looks messy.